src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Test/ScenarioTests/IaasVm/PolicyTests.cs

```diff
@@ -31,7 +31,11 @@ public PolicyTests(Xunit.Abstractions.ITestOutputHelper output)
             XunitTracingInterceptor.AddToContext(_logger);
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Backup tests require multiple versions of the same SDKs, not supported in NetStandard")]
```

Comment:
Let's file an issue for this - the solution is to update the tests (better) or to  compile the generated code for one of the SDKs into the solution

Reply:
Created: https://github.com/Azure/azure-powershell/issues/7533